staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -249,6 +249,13 @@ type ObjectMeta struct {
 	// This field is not set anywhere right now and apiserver is going to ignore it if set in create or update request.
 	// +optional
 	ClusterName string `json:"clusterName,omitempty" protobuf:"bytes,15,opt,name=clusterName"`
+
+	// LastApplied is a map of workflow-id (user-defined strings) to
```

Comment:
LastApplied is a map of workflow-id to the workflow's last applied configuration. A workflow is ____.